ckanext/rating/templates/rating/snippets/stars.html

```diff
@@ -20,10 +21,20 @@
 <span class="star-rating{% if stars == 0 %} no-stars{% endif %}">
   <span class="star-rating-stars">
     {%- for index in range(stars|int) -%}
-      <a class="icon icon-star rating-star" id="star-{{index}}" href="{{ h.url_for(controller='ckanext.rating.controller:RatingController', action=action, package=package.name, rating=index+1) }}"></a>
+      {% if ckan_29_or_higher %}
```

Comment:
Is this how it is used in other plugins also? this doesn't seem like a nice pattern :)